test/ejabberd_tests/tests/rest_SUITE.erl

```diff
@@ -62,15 +62,15 @@ groups() ->
 
 test_cases() ->
     [commands_are_listed,
-     non_existent_command_returns_404,
+     non_existent_command_returns404,
```

Comment:
Doesn't match standard regex for function names in Elvis. Segment after _ has to begin with letter.